gammapy/estimators/map/core.py

```diff
@@ -129,7 +129,7 @@ class FluxMaps:
         * stat_scan : optional, the test statistic scan values.
         * ts : optional, the delta TS associated with the flux value.
         * sqrt_ts : optional, the square root of the TS, when relevant.
-        * success : optional, a boolean tagging the validity of the estimation
+        * success : optional, a boolean tagging the validity of the estimation.
```

Comment:
Can you check the formatting here? 
It does not look right in the docs
https://docs.gammapy.org/1.1/api/gammapy.estimators.FluxMaps.html

Reply:
Solved thanks for checking!